azurelinuxagent/common/osutil/freebsd.py

```diff
@@ -22,13 +22,14 @@
 import azurelinuxagent.common.utils.fileutil as fileutil
 import azurelinuxagent.common.utils.shellutil as shellutil
 import azurelinuxagent.common.utils.textutil as textutil
-from azurelinuxagent.common.utils.networkutil import RouteEntry # pylint: disable=W0611
+from azurelinuxagent.common.utils.networkutil import RouteEntry  # pylint: disable=W0611
```

Comment:
Somehow I dont get these errors in pycharm (the extra space required ones)

Reply:
Interesting, I didn't modify any of the default settings around whitespace... I'm using the 2020.2 version

Reply:
I get those as well

Reply:
from PEP 8
    
    An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.

Reply:
Woah nice! I guess I need to fix my pycharm to actually complain about these :D